README.rst

```diff
@@ -1 +1,86 @@
-The api for the master facility list.
\ No newline at end of file
+==========
+Master Facility List API
+==========
+.. image:: https://circleci.com/gh/MasterFacilityList/mfl_api.svg?style=shield
+    :target: https://circleci.com/gh/MasterFacilityList/mfl_api
+
+This is the backend for the Master Facility list. 
+It provides an api through which one is able to access various resources provided by master facility list; Including facilities and their services.
+
+Read more at http://ehealth.or.ke/facilities/
+
+Getting started
+----------------
+**A: Running the system from source code**
+
+1. Create a virtualenv
+
+2. Activate the created vitualenv and run ``pip install -r requirements.``
+
+3. Run ``fab setup``
+    Running this command will do the following:
+
+    * Create the database.
+
+    * Run migrations.
+
+    * Load demo data if the project is in development mode.
+
+**B: Installing the system**
+
+
+2. Activate the virtualenv and run ``python setup.py install`` while in the project folder.
+
+Explore the api
+---------------
+Once all the steps above have been carried out, the system is ready to run and one can explore the api. 
+
+There are two ways to explore the api:
+
+**a. Through swagger**
```

Comment:
"Swagger" is a proper noun ( should start with a capital letter ).

You'd do well to also link to http://swagger.io/ - provide background for those that do not know what Swagger is.